### PR TITLE
Add installation guide to README for macos installation via `brew install oxker`

### PR DESCRIPTION
**Pull request by miketheman**

### CODE REVIEW

**Comment by miketheman** [comment]
Per https://formulae.brew.sh/formula/oxker

**Comment by mrjackwills** [comment]
Thanks, I don't have access to a Mac at the moment, is it working as expected on MacOS?

Also, do we have to do anything to keep brew in-sync with the GitHub/Crates.io version?

**Comment by mrjackwills** [comment]
I'm not sure if it necessarily needs a separate `Homebrew` section, it could just be in the `Download & install` section, maybe sandwiched between `Docker` and `Nix`

**Comment by miketheman** [comment]
@mrjackwills I added it basically the same as other installation methods. Happy to move it iof you think that's most important

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@
 </p>
 
 - [Download & install](#download--install)
+- [Homebrew](#homebrew)
```
Comment (miketheman) [inline README.md:19]: Don't need an entry at the top, for sure.
```suggestion
- [Homebrew](#homebrew)
```

**Comment by mrjackwills** [comment]
Looks good, will merge soon, there's a mistake in another section of the readme which has been bugging me and I need to fix.

**Pull request merged**